### ISSUE TITLE
Close option for Browse All blocks panel

### ISSUE THREAD

**Issue by webmaster-cob** [WordPress/gutenberg#25055]
1) Click the black Add Block button in the block editor
2) Click the Browse All button
3) A block panel opens on the left of the editor
4) There's no obvious way to close the panel

The only way we've figured out to close the browse all blocks panel is to click over in the body of the post on the Add Block button again. 

<img width="263" alt="2020-09-03 10_32_58-Edit Page ‹ City of Bellingham — WordPress and 8 more pages - Work - Microsoft​ " src="https://user-images.githubusercontent.com/26336976/92148309-3ae11a00-edd1-11ea-9cbf-c3e492cfa983.png">
<img width="286" alt="2020-09-03 10_34_43-Edit Page ‹ City of Bellingham — WordPress and 8 more pages - Work - Microsoft​ " src="https://user-images.githubusercontent.com/26336976/92148310-3b79b080-edd1-11ea-9a7a-f7055cd165e3.png">

**Comment by aristath** [comment]
As I understand it, when that panel opens, the assumption is that the user will either select a block, or continue what they were previously doing and just click on the main area to continue writing etc.
Worth discussing though as it may not be entirely clear. :+1:

**Comment by aristath** [comment]
Closing as a duplicate of https://github.com/WordPress/gutenberg/issues/22871
